arcgis-ios-sdk-samples/Display information/Query features with Arcade expression/QueryFeaturesArcadeExpressionViewController.swift

```diff
@@ -0,0 +1,111 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class QueryFeaturesArcadeExpressionViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+            // Set the touch delegate.
+            mapView.touchDelegate = self
+        }
+    }
+    
+    /// The arcade expression evaluation operation.
+    var evaluateOperation: AGSCancelable?
+    
+    /// Make and load a map.
+    func makeMap() -> AGSMap {
+        // Create a portal item with the portal and item ID.
+        let portalItem = AGSPortalItem(portal: .arcGISOnline(withLoginRequired: false), itemID: "14562fced3474190b52d315bc19127f6")
+        // Make a map with the portal item.
+        let map = AGSMap(item: portalItem)
+        // Load the map.
+        map.load() { _ in
+            // Set the visibility of all but the RDT Beats layer to false.
+            map.operationalLayers.forEach { layer in
+                let currentLayer = layer as? AGSLayer
+                if currentLayer?.name == "Crime in the last 60 days" || currentLayer?.name == "Police Stations" {
```

Comment:
Is it able to use the opposite condition, i.e., (and adjust the comment)

```swift
if let currentLayer = layer as? AGSLayer {
    currentLayer.isVisible = currentLayer.name.contains("RPD Beats")
}
```

Also I'm seeing a copy layer in the data. Should we remove it?

<img width="666" alt="image" src="https://user-images.githubusercontent.com/9660181/176062870-0894650a-2125-4c9b-9095-59b5fdbeb287.png">

Reply:
The data used is not managed by the Runtime account so I don't think we should alter the data.